Toggl.Ultrawave/Exceptions/ApiException.cs

```
@@ -48,10 +48,10 @@ private static string getLocalizedMessageFromResponse(IResponse response)
                 var error = JsonConvert.DeserializeObject<ResponseError>(response.RawData);
                 return error?.Message ?? badJsonLocalisedError;
             }
-            catch (JsonSerializationException)
-            {
-                return badJsonLocalisedError;
-            }
+            catch (JsonSerializationException) { }
```

btw. wasn't here from the getgo, why do we swallow exceptions here?